test/zmumu/tp_from_aod_MC.py

```diff
@@ -77,7 +77,8 @@
 process.muonMatchHLTL3.maxDeltaR = 0.1
 from MuonAnalysis.MuonAssociators.patMuonsWithTrigger_cff import *
 changeRecoMuonInput(process, "mergedMuons")
-#useExtendedL1Match(process) #MM no idea what the sequence did, not available since git migration
+#useL1Stage2Candidates(process)
+#appendL1MatchingAlgo(process)
```

Comment:
Also useL1Stage2Candidates in theory, though, as I don't know what @HuguesBrun  and @calderona plans are for updating the production of TnP ntuples in MC, I left them commented for now. I can uncomment if they want me to do that.